index.html

```diff
@@ -228,8 +228,25 @@ <h3 class="about-list"><span>Skills</span> <i class="fas fa-chevron-right"></i><
         </ul>
     </section>
 
-   
-   </main>   
-
+  
+  <footer class="contact-section">
+     <div class="contact-container">
+            <h2>Contact me</h2>
+          <div> 
+             <p>If you have an application you are interested in developing,
+              a feature that you need built or a project that needs coding.
+              I’d love to help with it</p> 
+            </div>
+        </div>
+        
+        <form action="/index.html" class="input-section" >
```

Comment:
- [ ] Hi, great work on adding the `action` attribute 👍. Although as per the requirements and the previous reviewer's statement you are going to have to implement the form using Formspree. 
![image](https://user-images.githubusercontent.com/18730106/144114579-fa5690a0-0a87-44b5-bb26-2c3fa344e914.png)
sign up to Formspree and they will give you the `form_id`
- [ ] Please kindly make sure the `method` attribute is declared as "POST".
- [ ] Please kindly make sure to check that using Formspree sends you an email when a user submits the form.